requirements/requirements.txt

```diff
@@ -218,12 +229,13 @@ twilio==6.10.4
 twisted==17.9.0           # via daphne
 txaio==2.9.0              # via autobahn
 typing==3.6.4             # via django-extensions, m2crypto
+urllib3==1.23             # via kubernetes
 uwsgi==2.0.17
 uwsgitop==0.10.0
-websocket-client==0.47.0  # via slackclient
+websocket-client==0.47.0  # via kubernetes, slackclient
 xmltodict==0.11.0
 zope.interface==4.4.3     # via twisted
 
 # The following packages are considered to be unsafe in a requirements file:
-pip==9.0.1
-setuptools==36.0.1
+# pip
```

Comment:
I don't know if this change is expected or not, this file is as output by the pip-compile instructions at the top of the file.